misc/test/performance_test.go

```diff
@@ -1,12 +1,10 @@
-//go:build Performance || all
-// +build Performance all
-
```

Comment:
I'm not familiar with how we build for the perf tests in this repo, but I assume we want to keep these.

Reply:
Yes indeed.  Thanks for catching this!